src/OSPSuite.R/Domain/SimulationBatch.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.R.Extensions;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.R.Domain
+{
+   public class SimulationBatchOptions
+   {
+      //Potentially null
+      public string[] VariableParameters { get; set; }
+
+      //Potentially null. This will be used when one parameter only is varied
+      public string VariableParameter { get; set; }
```

Comment:
Always issues with R (one vs multiple). I am creating two properties here so that we can either pass one or multiple parameter to vary